### DISCUSSION TITLE
Prevent animating initial mount

### DISCUSSION THREAD

**Discussion by nc** [motiondivision/motionone#62] [Support]
Hey – I have a component that is positioned using `transform: translate(x,y)`. I'd like to animate future changes to `x` and `y` but not the initial mount of the component. What's the best/most idiomatic way to achieve that with motionone?

**Comment by mattgperry** [comment]
What framework are you using it with?

**Reply by nc** [reply]
With React! I should have mentioned that.

**Reply by mattgperry** [reply]
Ah ok cool, is this the undocumented lib from this repo? I might need to double check but usually setting initial={false} will do this

**Reply by silas-v1** [reply]
Hi @mattgperry I am also looking for a solution for initial values to be used for the `motion` library for react.

for example:
```
 const ref = useRef(null);

  useEffect(() => {
    if (!ref?.current) return;

    animate(
      ref.current,
      {
        opacity: [0, 1],
        height: ["110px", "80px"],
      },
      {
        duration: 0.5,
      }
    );
  }, []);
```
I fail to find a way to set an initial value here
